pkg/image/origins.go

```diff
@@ -157,6 +157,7 @@ var OriginMap = map[string]string{
 	"mirrored-kiali-kiali":                                    "https://github.com/kiali/kiali",
 	"mirrored-kiwigrid-k8s-sidecar":                           "https://github.com/kiwigrid/k8s-sidecar",
 	"mirrored-kube-logging-logging-operator":                  "https://github.com/kube-logging/logging-operator",
+	"mirrored-kube-logging-fluentd":                           "https://github.com/kube-logging/fluentd",
```

Comment:
I'm getting 404 when accessing https://github.com/kube-logging/fluentd. Could the image be made in another repo perhaps?

Reply:
Oops, fixed :)